.github/workflows/tests.yml

```diff
@@ -14,6 +14,21 @@ jobs:
     - name: Generate documentation
       run: make docs
 
+  ClangFormat:
```

Comment:
For some reason this check was running on my PR before it got merged, and it was just stalling. Looking at my PR now it is no longer referenced, so it might have just been a GirHub bug? The action itself looks good.

Reply:
It was not a github bug. It was a user bug. Sleepy me put it in the required list before it was merged. I removed it before I merged your PR.